KioskTests/Bid Fulfillment/ConfirmYourBidPINViewControllerTests.swift

```diff
@@ -35,14 +34,10 @@ class ConfirmYourBidPINViewControllerTests: QuickSpec {
 
             customKeySubject.sendNext("2");
             expect(subject.pinTextField.text) == "2"
-
-            subject.keypadContainer.deleteCommand.execute(nil)
-            expect(subject.pinTextField.text) == ""
```

Comment:
I don't know how this (or the test that follows) _ever_ passed. It relies on listening to the `pinSignal` for updates, which we want because it should send "" when we invoke `deleteCommand`. **However**, we explicitly replace the subject's `pinSignal`, so it'll _never_ send "" :confused: